### ISSUE TITLE
Update example message for pt-BR

### ISSUE THREAD

**Issue by leeorf** [legend80s/commit-msg-linter#20]
Hello! First of all, thanks for this awesome lint.

I think that the example of a right formatted commit message for pt-BR language is wrong. I am brand new to open source contributing but I am looking forward to providing enough information to solve this problem, or even open the PR to fix the issue.

The thing is if we set the language to `pt-BR`, if we try to commit with the example provided in the error message, the error still persists due to first letter being capitalized.

For example, if I try to do this
```
git commit -m "teste"
```

It gives me this error message with the following:
```
exemplo: docs: Atualiza o README com link para a nova documentação
```

If I try to commit with the example (in Portugue "exemplo") message, I still can not commit because the first letter is capitalized.

**Comment by legend80s** [comment]
Yes it is a bug, I will correct it to "atualiza" tonight.

**Comment by legend80s** [comment]
corrected at 4.1.2
